app/network/entity/entity.form.component.js

```diff
@@ -229,5 +233,72 @@
     function storeGroups(groups) {
       _.setContents(entityForm.groups.items, groups);
     }
+
+    function incrementBulk() {
+      var is_bulk = entityForm.v4.is_bulk;
+      var increment_vlan = entityForm.v4.increment_vlan;
+      var ip_increment = entityForm.input.v4.increment_amount;
+
+      if (is_bulk) {
+        if (ip_increment) {
+          var wholeIP = entityForm.input.v4.address;
+          var ip = wholeIP.split('.');
+          var last = parseInt(ip.pop());
+          last += ip_increment;
+          ip.push(last);
+          entityForm.input.v4.address = ip.join('.');
+
+          entityForm.v4.range_end = (parseInt(entityForm.v4.range_end) + parseInt(ip_increment)).toString();
+
+          //nickname
+          var wholeName = entityForm.input.nickname;
+          ip = wholeName.split('/')[0].split('.');
+          last = parseInt(ip.pop());
+          last += ip_increment;
+          ip.push(last);
+          var end = wholeName.split('/')[1];
+
+          if (last === parseInt(last, 10) && end == parseInt(end, 10)) {
+            entityForm.input.nickname = ip.join('.')+'/'+wholeName.split('/')[1];
+          }
+
+          if (increment_vlan) {
+            entityForm.input.vlan++;
+
+            wholeIP = entityForm.input.v4.gateway;
```

Comment:
Since we're also modifying gateway here, we should mention that in the frontend ("Increment VLAN and Gateway").